apps/snp_store/snp_store.cpp

```diff
@@ -703,82 +734,161 @@ int detectSNPs(SNPCallingOptions<TSpec> &options)
                 return 1;
             }
         }
+    }
     // Allocate one BamAlignmentRecord for each input BAM file.
     std::vector<BamAlignmentRecord> records(length(options.readFNames));
 
     /////////////////////////////////////////////////////////////////////
+    std::time_t t = std::time(nullptr);                                         //Get time and date for vcf header
+    char now[24];
+    strftime(now, sizeof(now), "%Y%m%d", std::localtime(&t));
```

Comment:
Do you really write the timestamp of that file into the vcf file?
Is that common praxis and really necessary?
We can make tests with timestamps. At least not with the current system.
So you need make the timestamp writing optional (using a macro: like ```SEQAN_SNP_STORE_DISABLE_TIME_STAMP```) and let your cmake script add this, when ```SEQAN_BUILD_SYSTEM``` equals ```DEVELOP```, like:
```shell
if ("${SEQAN_BUILD_SYSTEM}" STREQUAL "DEVELOP")
   message (STATUS "Disabling time stamp feature for writing vcf files!")
   set (SEQAN_DEFINITIONS ${SEQAN_DEFINITIONS} -DSEQAN_SNP_STORE_DISABLE_TIME_STAMP)
endif ()
```
Thus you are able to fix the app tests.